### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by Jessemccallums**
Project requirements

I created the "header" section and the "container" section

I created the nav bar.

I created the ul il elements.

I created the hover for the button 

I Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.

I used template 4.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,86 +21,243 @@
...
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>Ruby</li>
+
+
+                </ul>
+
+                <button type="button" class="button1">See project</button>
+
+
+            </div>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] In this milestone, you are required to use `display: grid` to position these different projects. Please use display grid here 👍

File: index.html [hunk 2]
```diff
@@ -21,86 +21,243 @@
...
-               
+                <li class="styling2">
+                    <p>Work</p>
+                </li>
+
+
             </ul>
-        
+</section>
+            <section>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please ensure that the padding/margin around this section matches that of the Figma
### Your Implementation
image
### Figma
image

File: main.css [hunk 3]
```diff
@@ -124,3 +119,150 @@ ul {
...
+  padding-right: 35px;
+}
+
+.styling3 {
+  display: flex;
+  justify-content: space-around;
+  padding: 30px;
+}
+
+.button1 {
```
Comment (Lucas-Masaba) [inline main.css:201]: - [ ] This button does not have the pseudo classes such as hover as required by the template. Please add the pseudo classes as seen in the image below 🏄 
image

File: index.html [hunk 4]
```diff
@@ -21,86 +21,243 @@
...
+                </ul>
+
+                <button type="button" class="button1">See project</button>
+
+
+            </div>
+        </section>
+
+        <section class="image">
+            <h1 class="firsttext">CONTACT US</h1>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] The contact section here is not aligned similar to that of the Figma template. Please follow all the alignment as seen in the Figma
### Your Implementation
image
### Figma
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Jessemccallums  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Professsional README 📓 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Jessemccallums) [reply index.html]: Hi Sir, 

This part of the project is not required yet, I added it by accident.
So I need to remove it.
Hope you understand, thank you!

Best regards.

-Jesse McCallums

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Jessemccallums  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Implemented requested changes 🏄 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**